kustomize/components/cymbal-branding/kustomization.yaml

```diff
@@ -14,18 +14,18 @@
 
 apiVersion: kustomize.config.k8s.io/v1alpha1
 kind: Component
-patchesStrategicMerge:
-# frontend - enable CYMBAL_BRANDING
-- |-
-  apiVersion: apps/v1
-  kind: Deployment
-  metadata:
-    name: frontend
-  spec:
-    template:
-      spec:
-        containers:
-          - name: server
-            env:
-            - name: CYMBAL_BRANDING
-              value: "true"
+patches:
+  # frontend - enable CYMBAL_BRANDING
+  - |-
```

Comment:
there is extra indentation that breaks the format. Please shift left _all_ content under `patches`